Sofa/framework/Config/CMakeLists.txt

```
@@ -259,6 +259,12 @@ if (SOFA_TRACY)
     message(STATUS "SOFA is compiled with the Tracy profiler client. Use the Tracy server ${SOFA_TRACY_VERSION}.")
 endif()
 
+
+option(SOFA_SCOPED_ADVANCED_TIMER "Compile SOFA with the scoped advanced timer" ON)
+if(SOFA_SCOPED_ADVANCED_TIMER)
+    set(ENABLE_SCOPED_ADVANCED_TIMER 1)
```

```suggestion
    set(SOFA_ENABLE_SCOPED_ADVANCED_TIMER 1)
```
usually, "global" macro variable like that has `SOFA_` as a prefix